src/rules/property-blacklist/README.md

```diff
@@ -0,0 +1,55 @@
+# property-blacklist
+
+Specify a blacklist of disallowed properties.
+
+```css
+    a { 
+      color: pink;
+    }
+/**    ↑ 
+ * These properties */
+```
+
+## Options
+
+`array`: `"["array", "of", "unprefixed", "properties"]`
+
+### `["array", "of", "unprefixed", properties"]`
+
+Blacklisted properties *must never* be used.
+
+Given:
+
+```js
+["background-size", "transform"]
```

Comment:
Is there any more realistic set of properties that we could use?

(I haven't thought much about this one because I haven't been sure about the real-world application of it.)